rules/60-cdrom_id.rules

```diff
@@ -8,6 +8,10 @@ ENV{DEVTYPE}!="disk", GOTO="cdrom_end"
 # unconditionally tag device as CDROM
 KERNEL=="sr[0-9]*", ENV{ID_CDROM}="1"
 
+# stop automatically any mount units bound to the device if the media eject
+# button is pressed.
+ENV{ID_CDROM}=="1", ENV{SYSTEMD_BIND_MOUNTS}="1"
```

Comment:
hmm, "BIND_MOUNTS" suggests relationship with the kernel "bind mounts" feature (i.e. mount --bind and stuff), let's use the same naming here as for the new mount option, and call this SYSTEMD_MOUNT_DEVICE_BOUND=1 or so?